packages/twenty-front/src/modules/object-record/record-filter/utils/computeViewRecordGqlOperationFilter.ts

```diff
@@ -648,6 +710,35 @@ const computeFilterRecordGqlOperationFilter = (
       }
       break;
     }
+    case 'ARRAY': {
+      switch (filter.operand) {
+        case ViewFilterOperand.Contains:
+          return {
+            [correspondingField.name]: {
+              containsIlike: filter.value, // Why doesn't `%${filter.value}%` here? %-signs only work when they are in compute-where-condition-parts.ts.
+            } as StringFilter,
+          };
+        case ViewFilterOperand.DoesNotContain:
+          return {
+            not: {
+              [correspondingField.name]: {
+                containsIlike: `%${filter.value}%`,
+              } as StringFilter,
```

Comment:
I don't understand this as StringFilter, the whole thing should be an arrayFilter. Not sure why we have all these as too but that's another story